packages/ruleset/src/functions/api-symmetry.js

```diff
@@ -154,283 +151,230 @@ function checkForGraphFragmentPattern(
+      // that don't exist on the canonical schema.
+      if (!valid && !(considerWriteOnly && prop.writeOnly)) {
         infoLogStack.push(
-          `${ruleId}: variant schema applicator '${applicator}' is not a graph fragment of the canonical schema`
+          propExistsSomewhere
+            ? `${ruleId}: nested object property ${name} is not a graph fragment of canonical property ${name}`
+            : `${ruleId}: property '${name}' does not exist on the canonical schema`
         );
         result = false;
       }
-    });
-
-    // To be a proper graph fragment, at least one property needs to be omitted
-    // from the schema structure. Check to make sure this is the case.
-    //
-    // For this, variant schemas need to be taken as a whole (as opposed to
-    // looking at each applicator schema individually, if they exist) so  we
-    // skip this check if we're currently processing a variant applicator
-    // schema.
-    //
-    // If we've already determined that the variant schema omits at least one
-    // property, we don't need to re-perform this check anymore.
-    //
-    // Note: we don't bother checking for reference schemas here because there
-    // should be differences other than omitting non-reference properties.
-
-    if (!fromApplicator && !variantOmitsProperties) {
-      logger.debug(
-        `${ruleId}: checking that variant schema is a proper subset of canonical schema`
-      );
-
-      const propertyOmitted = schemaHasConstraint(canonical, c => {
-        if (c && isObject(c.properties)) {
-          let omitted = false;
-          for (const [name, prop] of Object.entries(c.properties)) {
-            logger.debug(`${ruleId}: checking canonical property '${name}'`);
-
-            const included = schemaLooselyHasConstraint(
-              variant,
-              s =>
-                'properties' in s &&
-                isObject(s.properties[name]) &&
-                getSchemaType(s.properties[name]) === getSchemaType(prop)
-            );
-
-            logger.debug(
-              `${ruleId}: finished checking canonical property '${name}', included=${included}`
-            );
-
-            if (!included) {
-              omitted = true;
-              logger.debug(
-                `${ruleId}: canonical property ${name} is NOT present in variant schema`
-              );
-              break;
-            } else {
-              logger.debug(
-                `${ruleId}: canonical property ${name} is present in variant schema`
-              );
-            }
-          }
-
-          return omitted;
-        }
-        return false;
-      });
-
-      if (propertyOmitted) {
-        variantOmitsProperties = true;
-      }
-      logger.debug(
-        `${ruleId}: finished checking for omitted properties, propertyOmitted=${propertyOmitted}`
-      );
-    } else {
-      logger.debug(`${ruleId}: bypassing omitted properties check`);
```

Comment:
This is the actual change. All of this code was removed as no longer correct per changes in the API Handbook.